contracts/container/contract.go

```diff
@@ -106,8 +106,11 @@ const (
 	nnsRootKey         = "nnsRoot"
 	nnsHasAliasKey     = "nnsHasAlias"
 
-	corsAttributeName = "CORS"
-	lockAttributeName = "__NEOFS__LOCK_UNTIL"
+	corsAttributeName            = "CORS"
+	lockAttributeName            = "__NEOFS__LOCK_UNTIL"
+	tagsAttributeName            = "TAGS"
```

Comment:
It's S3_TAGS for sure. For all other intents and purposes container attributes are tags already.